qa/rbd/krbd_blkroset.t

```diff
@@ -2,8 +2,11 @@
 Setup
 =====
 
+  $ RO_KEY=$(ceph auth get-or-create-key client.ro mon 'profile rbd' mgr 'profile rbd' osd 'profile rbd-read-only')
```

Comment:
Nit: the `mgr 'profile rbd'` code hasn't been backported to Nautilus (yet). It looks like it should just be safe to drop that specific cap since this test doesn't issue any MGR commands.

Reply:
It seems mgr had been pretty lax about profiles until your changes to fix "rbd rm" offload went in.  This set of caps was copied from the reporter's email on ceph-users, they are on 14.2.2.  I've also tested this PR against the nautilus branch, it passes.

IIUC the backport of `mgr 'profile rbd'` code won't change anything with respect to this test script, so I don't see the reason for dropping mgr cap in this backport.

Reply:
Ah, good point -- it will be ignored until it just starts doing something.